src/OpenTelemetry.ResourceDetectors.ProcessRuntime/OpenTelemetry.ResourceDetectors.ProcessRuntime.csproj

```
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry Extensions - Process Runtime Resource Detector from .NET runtime.</Description>
```

This might be misleading as this package is not developed by the .NET team.

```suggestion
    <Description>OpenTelemetry Extensions - Process Runtime Resource Detector for .NET runtime.</Description>
```